docs/contributing/release_guide.rst

```diff
@@ -117,49 +101,10 @@ Perform code quality checks
 Create the release branch
 =========================
 
-* Enter the :file:`PlasmaPy` directory and create a new branch for the
-  release that is based off of the ``main`` branch. For a bugfix
-  release, this branch should already exist.
-
-  .. code-block:: bash
-
-     git checkout -b v0.9.x upstream main
-
-  The ``upstream`` remote corresponds to `PlasmaPy's GitHub repository`_.
-
-* Push the branch to `PlasmaPy's GitHub repository`_.
-
-  .. code-block:: bash
-
-     git push -u upstream
-
-* Open a pull request to transform the news fragments in
-  :file:`changelog/` to a changelog page.
-
-  - In the top-level directory, run:
-
-    .. code-block:: bash
-
-       towncrier build --version 0.9.0
-
-    When asked about removing changelog entries, do so.
-
-  - Copy the relevant parts of the generated :file:`CHANGELOG.rst` file
-    into :file:`docs/whatsnew/0.9.0.rst`.
-
-  - Add the entry for :file:`docs/whatsnew/0.9.0.rst` in the table of
-    contents in :file:`docs/whatsnew/index.rst`.
-
-    .. todo::
-
-        Immediately following the ``v0.8.1`` release, we made (or
-        planned to make) a few changes to the towncrier_ setup
-        (:pr:`1623`, :pr:`1626`, :issue:`1627`). This guide may require
-        some updates for the subsequent release.
-
-    .. todo::
-
-       We might be able to consolidate these steps into a single one.
```

Comment:
I was considering whether it might not make sense to keep this somewhere in case the automation breaks or I get hit by a bus (less likely with remote work nowadays, but still!) - but then I realized that hey, we've got git for that! :D